labs/multiverse/src/main.rs

```diff
@@ -448,6 +456,29 @@ impl App {
                             }
                         }
                     }
+                    GlobalMode::Searching { view } => {
+                        if let Event::Key(key) = event
+                            && let KeyModifiers::NONE = key.modifiers
+                        {
+                            match key.code {
+                                Enter => {
+                                    if let Some(query) = view.get_text() {
+                                        if let Some(room) = self.room_view.room() {
+                                            if let Some(results) = room.search(&query, 100).await {
+                                                view.results(results);
+                                            } else {
+                                                println!("sadge no results")
```

Comment:
Is `println!` the right way to log? I see `error` above, so maybe `warn` would be better here?